### PR TITLE
Making code working with older version of rails/rspec

### PR DESCRIPTION
**Pull request by shvets**
Hi,

I provided fix for your code to work with older version of rails(1.8.x) and rspec(1.3.x). Actually, this code will work with newer versions too. 

Also, in fork scope, your 'files' object is array, but rspec require it as a string. I use *files to fix the problem.

### CODE REVIEW

File: bin/spin [hunk 1]
```diff
@@ -40,7 +40,7 @@ def serve(force_rspec = false)
 
   ENV['RAILS_ENV'] = 'test' unless ENV['RAILS_ENV']
 
-  if File.exist? 'config/application.rb'
+  if File.exist? 'config/environment.rb'
```
Comment (jstorimer) [inline bin/spin:43]: If you read the comment just below this line you'll know why we don't load config/environment.rb at this stage. That will be the case for any version of Rails.

**Comment by jstorimer** [comment]
I don't really have an interest in supporting the 1.x series of Rails. I'd be ok with supporting the 2.3.x series (and if that inadvertently also support the 1.x series then that's fine).

I'd consider merging support for RSpec 1.x but the changes you made are not forward compatible with RSpec 2.x. Can you make it compatible with both versions?

**Comment by shvets** [comment]
under rails 1.8.7 there is no application.rb file. RAILS_GEM_VERSION
variable can separate code for different releases.

2011/11/3 Jesse Storimer <
reply@reply.github.com>

**Comment by jstorimer** [comment]
The discussion going on in #5 is the direction we'll take in supporting older versions of Rails.

**Comment by shvets** [comment]
This should fix:

```
    if Spec::VERSION::STRING =~ /^1/
      ARGV.push *files
      require 'spec/autorun'
    else
      ARGV.push files
      require 'rspec/autorun'
    end
```

2011/11/3 Jesse Storimer <
reply@reply.github.com>

**Comment by jstorimer** [comment]
This won't work because RSpec 2.x doesn't define a `Spec` constant. I get `NameError: uninitialized constant Spec`.

**Comment by shvets** [comment]
you can use

require 'rspec/core/version'

RSpec::Core::Version::STRING

for 2.x branch

2011/11/8 Jesse Storimer <
reply@reply.github.com>

**Pull request closed without merging**